pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +44,8 @@ import (
 
 const ResourceResyncTime = 60 * time.Second
 
+const OrphanFinalizerID = "orphanFinalizer"
```

Comment:
Custom finalizers should have a prefix assuming finalizers are qualified names (like namespaces).  So if you write a custom one it needs to be mycompany.org/mything. We should reserve the unqualified space for Kubernetes.  So validation would check that non-standard finalizers were qualified.  Just like resources....